root/static/scripts/release-editor/fields.js

```diff
@@ -387,6 +398,11 @@ class Track {
     return capitalizedETIRegex.test(this.name());
   }
 
+  hasMiscapitalizedTitle() {
+    return this.medium.release.shouldUseEnglishCapitalization() &&
+      miscapitalizedEnglishRegex.test(this.name());
```

Comment:
One potential problem that I noticed is that this function (and also `hasFeatInTitle` and `hasCapitalizedETI`) run for every keystroke while the user is typing into a title input. I think that it's happening due to `Track`'s `name` member being updated on `input` rather than `change`.

This could have performance implications, and it might also be annoying from a UX perspective (although I think that all three checks are unlikely to be triggered incorrectly mid-title). I tried working around it by wrapping these functions in `debounceComputed`, but that has the downside of making them to be slower to give feedback, and it also means that the warnings take half a second to disappear after the user clicks "Guess feat." or "Guess case" to fix the titles.

Reply:
I think I'd be happier with debouncing if there were some way to trigger evaluation immediately after "Guess case" et al is clicked, but it wasn't obvious to me from [the Knockout docs](https://knockoutjs.com/documentation/rateLimit-observable.html) whether that's possible.

I was worried at first about warnings showing up for e.g. "The" while the user is manually typing "There", but I think that that's unlikely due to the regexes looking for trailing whitespace after the error.

Reply:
What about specifying a shorter debounce interval (second argument to `debounceComputed`)? It defaults to 500ms, but 250ms is probably enough to avoid updates between keystrokes while reducing the update lag after hitting the guess-case button too.

Reply:
250 ms feels pretty good to me; thanks for the suggestion. I've also updated the methods for feat. in titles and capitalized ETI to be debounced.

Reply:
When this runs and it triggers the note, I guess it will shift the page down a bit? It could be annoying in terms of UX, including with the delay.
But my overall impression is that not having to fix endless #^%ing Discogs imports is more than worth it :)
I think this would best be tweaked after feedback in beta - though beginner users might not give feedback, which is a different issue.

Reply:
@Aerozol, yeah, there's unfortunately a visible shift. I couldn't think of any way to avoid it apart from reserving empty space at the top of the page (ugh) or displaying the message somewhere else (which may require a fair amount of re-engineering; I just piggybacked on the current warning-displaying logic for this).

I'm hoping it won't be a huge deal, since (barring false positives) it'll be displayed when people aren't following the guidelines, plus most editors seem to be seeding releases now so the warning should already be there when they switch to the tracklist tab. The shift might be weird if someone is manually typing titles and has entered enough tracks to scroll the screen down so they can't see the warning, though...